docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -12,7 +12,7 @@ We then walk you through building your first {project-full-name} application, di
 == Introducing Spring Cloud Sleuth
 
 Spring Cloud Sleuth provides API for distributed tracing solution for https://cloud.spring.io[Spring Cloud].
-It integrates out of the box with two tracer implementations:
+It integrates out of the box with tracer implementation:
 
 * https://github.com/openzipkin/brave[OpenZipkin Brave]
```

Comment:
Then you can delete this line (and close up the blank lines as needed).

Reply:
Done @Buzzardo